library/ssl_tls.c

```diff
@@ -333,7 +333,6 @@ static void handle_buffer_resizing( mbedtls_ssl_context *ssl, int downsizing,
 }
 #endif /* MBEDTLS_SSL_VARIABLE_BUFFER_LENGTH */
 
-#if defined(MBEDTLS_SSL_PROTO_TLS1_2)
```

Comment:
I doubt we use the two functions below in TLS 1.3 thus it would be better to keep the pre-compiler directive "#if defined(MBEDTLS_SSL_PROTO_TLS1_2)" here I'd say.